.travis.yml

```diff
@@ -4,6 +4,7 @@
 # Available here:
 #   - http://felixduvallet.github.io/ros-travis-integration
 #   - https://github.com/felixduvallet/ros-travis-integration
+#   - https://github.com/gsilano/ros-travis-integration
```

Comment:
Thanks for submitting this Pull Request!
There shouldn't be a need for an additional location once this gets merged in. Please remove this line.

Reply:
Ok! I removed it!